test

```diff
@@ -40,6 +40,8 @@ function clean()
         (cd $path_test && ws destroy)
         rm -rf $path_test
     fi
+
+    rm -f ./chart
```

Comment:
what creates this? just wounderign why it's not something created in .tmp-test

Reply:
I think it's created by `ws helm kubeval qa`